meta-cli/src/typegraph/postprocess.rs

```diff
@@ -111,7 +149,9 @@ pub mod deno_rt {
                 let mut mat_data: ModuleMatData = object_from_map(std::mem::take(&mut mat.data))?;
                 if !mat_data.code.starts_with("file:") {
                     // TODO check imported functions exist
-                    mat_data.code = transform_module(mat_data.code)?;
+                    let path = mat_data.code.strip_prefix("file:").unwrap();
```

Comment:
@Natoandro is there a way to remove the file?

Reply:
We can remove it now.